packages/node_modules/@webex/internal-plugin-wdm/src/index.js

```diff
@@ -2,28 +2,21 @@
  * Copyright (c) 2015-2019 Cisco Systems, Inc. See LICENSE file.
  */
 
-import {registerInternalPlugin} from '@webex/webex-core';
-
-import Device from './device';
-import config from './config';
-import DeviceUrlInterceptor from './interceptors/device-url';
-
-registerInternalPlugin('device', Device, {
-  interceptors: {
-    DeviceUrlInterceptor: DeviceUrlInterceptor.create
-  },
+import Device, {
```

Comment:
With this type of import/export setup, the `@webex/internal-plugin-wdm` simply points to the new `@webex/internal-plugin-device` so that they are identical in every way. This prevents the need to update every import across all packages.

Reply:
I think this works for now, but we should probably completely remove usage at some point (sooner rather than later)